plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesPluginImpl.kt

```diff
@@ -988,13 +997,14 @@ class GesturesPluginImpl : GesturesPlugin, GesturesSettingsBase {
       duration = animationTime
     }
 
-    zoomAnimator.addListener(object : AnimatorListenerAdapter() {
+    anchorAnimator.addListener(object : AnimatorListenerAdapter() {
 
       override fun onAnimationStart(animation: Animator) {}
 
       override fun onAnimationCancel(animation: Animator) {}
 
       override fun onAnimationEnd(animation: Animator) {
+        cameraAnimationsPlugin.resetAnchor(cachedAnchor)
```

Comment:
What if the zoom gesture velocity animation was canceled by a developer's anchor animation?

I guess the cancelation happens first, so we will first reset the anchor to the cached value, and then the developer's transition will proceed so the end result should be correct, right?

Reply:
@LukasPaczos yes, it will work like this - we would first cancel, it will also trigger an end callback, we will reset anchor and start user anchor animation with cached value that he was actually expecting.